x/cert/simulation/decoder.go

```diff
@@ -50,6 +50,24 @@ func DecodeStore(cdc *codec.Codec, kvA, kvB tmkv.Pair) string {
 		cdc.MustUnmarshalBinaryLengthPrefixed(kvB.Value, &certifierB)
 		return fmt.Sprintf("%v\n%v", certifierA, certifierB)
 
+	case bytes.Equal(kvA.Key[:1], types.NextCertificateIDKey()):
+		var idA, idB uint64
+		cdc.MustUnmarshalBinaryLengthPrefixed(kvA.Value, &idA)
+		cdc.MustUnmarshalBinaryLengthPrefixed(kvB.Value, &idB)
+		return fmt.Sprintf("%v\n%v", idA, idB)
+
+	case bytes.Equal(kvA.Key[:1], types.CertifierCertIDsStoreKeyPrefix):
+		var idsA, idsB []uint64
+		cdc.MustUnmarshalBinaryLengthPrefixed(kvA.Value, &idsA)
+		cdc.MustUnmarshalBinaryLengthPrefixed(kvB.Value, &idsB)
+		return fmt.Sprintf("%v\n%v", idsA, idsB)
+
+	case bytes.Equal(kvA.Key[:1], types.ContentCertIDStoreKeyPrefix):
```

Comment:
They can be merged when they are doing the same thing:
```
case case1, case2, case3:
    ...
```

Reply:
Oh I see. Thanks.